### DISCUSSION TITLE
Notification

### DISCUSSION THREAD

**Discussion by mailtodillibabu** [amborle/featmap#24] [Ideas]
It will be good if we have the notification for further changes it will notify all the people involved in the storyboard.
Just thought it will be valuable for users.

**Comment by amborle** [comment]
What kind of changes would you like notifications for? Lots of things can change in a story board: new features/releases/goals, changes of features/releases goals, (re-)prioritizations, comments etc etc. 

Should the notifications be off by default and the user needs to activate notifications per story board?

Please share some more details and use cases, please 😀

**Reply by mailtodillibabu** [reply]
Thanks for considering my suggestion/idea.

**Use Cases:** Any one of the below use cases maybe suitable for you.
1. As per the screenshot that I have shared below, Under David & Rebbaca there are three stories **Find book, Search for book & View details**. For any update/action on that 3 stories, it has to be notified by both David & Rebbeca.
**For Example:** If the **Find book** story has been completed & marked as done, then David & Rebbeca has to be notified of this with further email/slack/anyone client.

2. We need to have **Add Followers/Follow** option for a storyboard with their email ids who are all involved. Whatever the updates/action taken in that storyboard they will receive further notifications.
**For Example:** Github **watch list**.

![Screenshot from 2021-02-28 09-57-33](https://user-images.githubusercontent.com/58637501/109407988-7a1cac00-79ab-11eb-8b21-ac8b217dcdc6.png)
